internal/backends/qt/qt_widgets/lineedit.rs

```diff
@@ -80,8 +83,12 @@ impl Item for NativeLineEdit {
     ) -> LayoutInfo {
         LayoutInfo {
             min: match orientation {
-                Orientation::Horizontal => self.native_padding_left() + self.native_padding_right(),
-                Orientation::Vertical => self.native_padding_top() + self.native_padding_bottom(),
+                Orientation::Horizontal => {
+                    (self.native_padding_left() + self.native_padding_right()).get()
+                }
+                Orientation::Vertical => {
+                    (self.native_padding_top() + self.native_padding_bottom()).get()
+                }
             },
```

Comment:
(style) I wonder if this works?

```suggestion
                Orientation::Horizontal => self.native_padding_left() + self.native_padding_right(),
                Orientation::Vertical => self.native_padding_top() + self.native_padding_bottom(),
            }.get(),
```

(just to save a few lines :-p)

Reply:
Good idea. I'll apply that throughout.